feature-libs/my-account/organization/src/components/shared/organization.model.ts

```diff
@@ -1,5 +1,10 @@
 export enum OrganizationTableType {
   COST_CENTER = 'costCenter',
   COST_CENTER_BUDGETS = 'costCenterBudgets',
-  COST_CENTER_ASSIGN_BUDGET = 'costCenterAssignBudget',
+  COST_CENTER_ASSIGN_BUDGET = 'costCenterAssignBudgets',
+  USER_GROUP = 'userGroup',
+  USER_GROUP_USERS = 'userGroupUsers',
+  USER_GROUP_ASSIGN_USER = 'userGroupAssignUsers',
```

Comment:
For consistency, can we align the keys with the values (i,e. key ....USER vs value ...Users).
Same for COST_CENTER_ASSIGN_BUDGET